diagnostics-app/src/app/events/events.module.ts

```diff
@@ -1,16 +1,20 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 
+import { EventsCardComponent } from './events-card/events-card.component';
 import { EventsCardContentComponent } from './events-card/events-card-content/events-card-content.component';
 import { EventsDashboardComponent } from './events-dashboard/events-dashboard.component';
+import { SharedModule } from '../shared/shared.module';
```

Comment:
Do we need this module?

Reply:
Yep. This is related to all the rendering of material UI modules.

Reply:
Ahh got it. Thanks for explanation!